cockroachcloud/export-metrics.md

```diff
@@ -0,0 +1,264 @@
+---
+title: Export Metrics From a CockroachDB dedicated Cluster
+summary: Export Metrics From a CockroachDB dedicated Cluster
+toc: true
+docs_area: manage
+---
+
+{{ site.data.products.dedicated }} users can use the [Cloud API](cloud-api.html) to configure metrics export to [AWS CloudWatch](https://aws.amazon.com/cloudwatch/) or [Datadog](https://www.datadoghq.com/). Once the export is configured, metrics will flow from all nodes in all regions of your {{ site.data.products.dedicated }} cluster to your chosen cloud metrics sink.
+
+The metrics export feature to AWS CloudWatch is only available on {{ site.data.products.dedicated }} clusters created after August 11, 2022. Metrics export to Datadog is supported on all {{ site.data.products.dedicated }} clusters regardless of creation date.
+
+{% include feature-phases/preview-opt-in.md %}
+
+## The `metricexport` endpoint
+
+To configure and manage metrics export for your {{ site.data.products.dedicated }} cluster, use the `metricexport` endpoint:
+
+{% include_cached copy-clipboard.html %}
+~~~
+https://cockroachlabs.cloud/api/v1/clusters/{your_cluster_id}/metricexport
+~~~
+
+Access to the `metricexport` endpoint requires a valid {{ site.data.products.db }} [service account](console-access-management.html#service-accounts) with the appropriate permissions.
+
+The following methods are available for use with the `metricexport` endpoint, and require the listed service account permissions:
+
+Method | Required permissions | Description
+--- | --- | ---
+`GET` | `ADMIN`, `EDIT`, or `READ` | Returns the current status of the metrics export configuration.
+`POST` | `ADMIN` or `EDIT` | Enables metrics export, or updates an existing metrics export configuration.
+`DELETE` | `ADMIN` | Disables metrics export, halting all metrics export to AWS CloudWatch or Datadog.
+
+See [Service accounts](console-access-management.html#service-accounts) for instructions on configuring a service account with these required permissions.
+
+## Enable metrics export
+
+<div class="filters clearfix">
+  <button class="filter-button" data-scope="aws-metrics-export">AWS CloudWatch</button>
+  <button class="filter-button" data-scope="datadog-metrics-export">Datadog</button>
+</div>
+
+<section class="filter-content" markdown="1" data-scope="aws-metrics-export">
+
+{{site.data.alerts.callout_danger}}
+The metrics export feature is only available on {{ site.data.products.dedicated }} clusters created after August 11, 2022. If your {{ site.data.products.dedicated }} cluster was created before this date, you must create a new cluster to utilize metrics export, or [export metrics to Datadog](export-metrics.html?filters=datadog-metrics-export) instead.
```

Comment:
nit: since we're working on backfilling these missing service accounts to solve this issue, maybe it's worth noting that this is only temporary?

Reply:
Sounds good!